ci/scale-pure-ovn-hosts.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Save trace setting
+XTRACE=$(set +o | grep xtrace)
+set -o xtrace
+
+# Read variables
+source ovn-scale.conf
+
+# Library files
+source scale-lib.sh
```

Comment:
We could, but I run these individually sometimes, so I'd rather leave it in each file, similar to line 8 where it sources the .conf file as well.